WIP: Add documentation per fragment (#442)
okkur/syna: exampleSite/content/fragments/nav/docs.md

+++
fragment = "content"
weight = 300

title = "Documentation"

[sidebar]
  enable = true
+++

### Menus

- .Site.Menus.main

### Variables

#### search
*type: boolean*  
*default: false*

> If set to `true`, a text input would be added to the navbar and changes the layout of menus. The input is used for search.

Reviewer: ```suggestion
If set to `true`, search is enabled within the navbar.  

**Note:** The additional input field used for search will alter the menu layout.
```

#### prepend, postpend
*type: array of objects*

Menu like objects that are used to add menus before and after the main menu.

##### prepend/postpend.url
*type: string*

##### prepend/postpend.name
*type: string*

#### breadcrumb
*type: object*

If object is set, breadcrumbs will be shown under the navbar.

##### breadcrumb.display

> *type: boolean*

Reviewer: ```suggestion
*type: boolean*  
```

> *default: in case object is defined the value is true, if not, it's false*

Reviewer: What are you trying to say?

Author: Now that I think about it, it's unnecessary. I was trying to say that breadcrumb wouldn't display unless the object is defined.

##### breadcrumb.level

> *type: number*

Reviewer: ```suggestion
*type: number*  
```

*default: 1*

> Specifies from which level of nested directories the breadcrumb should be shown. For example for the value of 1:

Reviewer: ```suggestion
Define the section level the breadcrumb will start being shown.
 
The default value `1` would lead to the following being defined: 
```

```
content/_index # level 0, not shown
content/blog/_index # level 1, shown
content/blog/article-1 # level 2, shown
```

##### breadcrumb.background

> *type: primary, secondary, light, dark, warning, success (other Bootstrap background colors are acceptable as well)*  

Reviewer: ```suggestion
*type:  string*  
*recommended: primary, secondary, white, light, dark*  
*accepted values: primary, secondary, white, light, dark, warning, success, danger, info, transparent*
```

*default: light*

#### asset
*type: asset object*

> Asset will be shown as logo.

Reviewer: ```suggestion
Asset will be shown as a clickable logo directing to the main page.
```

Reviewer: Is that actually always the case? Do we overwrite this, when asset.url is provided?

Author: It's always overwriting the url.

#### repo_button
*type: object*

> If set, a repo button would be added to the navbar much like the Github button on top of this page.

Reviewer: ```suggestion
Enable a button on the top right navbar. Usually used to link to your repository such as Github or Gitlab.  
The icon can be customized via `repo_button.icon`.
```
